src/NuGet.Core/NuGet.Build.Tasks.Console/Program.cs

```diff
@@ -7,7 +7,9 @@
 using System.Globalization;
 using System.IO;
 using System.Linq;
+#if NETFRAMEWORK
```

Comment:
Can you explain me what is this change doing?

Reply:
This namespace is only needed for .NET Framework so when you build for .NET Core in VS this line has a red squiggly so this just cleans that up